pkg/ec2helper/ec2helper.go

```diff
@@ -909,11 +912,39 @@ func (h *EC2Helper) ParseConfig(simpleConfig *config.SimpleInfo) (*config.Detail
 		}
 	}
 
+	// Add simple-ec2 tags to created resources
+	simpleEc2Tags := getSimpleEc2Tags()
+	if simpleConfig.UserTags != nil {
+		for _, rawTag := range simpleConfig.UserTags {
+			kvSlice := strings.Split(rawTag, ":")
```

Comment:
1) this can be simplified by using a map input flag and `:` can be a valid char for tag value. 

2) `:` in tags is quite common. See valid chars [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions). 
For example, see tags attached by CFN. `The aws: prefix is reserved for AWS use.`